{{ cookiecutter.app_name }}/{{ cookiecutter.app_name }}/tests/test_views.py

```diff
@@ -0,0 +1,21 @@
+"""
+Test end to end django views.
+"""
+from django.test import TestCase
+from django.test.client import Client
+from django.core.urlresolvers import reverse
+
+
+class TestViews(TestCase):
+    """
+    Test that the views work as expected.
+    """
+    def setUp(self):
+        """Common test setup"""
+        super(TestViews, self).setUp()
+        self.client = Client()
+
+    def test_index_view(self):
+        """Verify the index view is as expected"""
+        response = self.client.get(reverse('{{ cookiecutter.app_name }}_index'))
+        self.assertContains(response, 'Hi', status_code=200)
```

Comment:
Nit: A two-letter word seems a bit flimsy to assert happiness, although you could argue the 200 alone is enough.